files/scripts/syncd_common.sh

```diff
@@ -163,7 +163,11 @@ stop() {
 
     stopplatform1
 
-    /usr/bin/${SERVICE}.sh stop $DEV
+    if [[ x"$WARM_BOOT" == x"true" ]]; then
+        /usr/bin/${SERVICE}.sh stop $DEV &
```

Comment:
why this goes to background ?

Reply:
to warm boot time as explained in the whole thread